kedro/templates/project/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/pipelines/example/pipeline.py

```diff
@@ -0,0 +1,10 @@
+from kedro.pipeline import Pipeline, node
+from .nodes import create_data, process_data
+
+def create_pipeline(**kwargs):
+    return Pipeline(
+        [
+            node(func=create_data, inputs=None, outputs="raw_data", name="create_data_node"),
+            node(func=process_data, inputs="raw_data", outputs="processed_data", name="process_data_node"),
+        ]
```

Comment:
Yeah that's a question I was asking myself as well. Having no example nodes would cause the test to fail, but I'm not sure if that would be that undesirable, since it's supposed to be a code example. 

Another option would be mocking the response instead of using an actual node, but that would go against the "no mocking" idea that was discussed on the issue.